README.md

```diff
@@ -33,27 +33,32 @@ which Chrono builds upon and should acknowledge:
 
 ## Limitations
 
-Only proleptic Gregorian calendar (i.e. extended to support older dates) is supported.
```

Comment:
Please leave out unrelated changes to reflow text, which mostly make it harder to review this.